plugins/package-managers/conan/src/main/kotlin/ConanV1Handler.kt

```diff
@@ -71,11 +71,11 @@ internal class ConanV1Handler(private val conan: Conan) : ConanVersionHandler {
             ).requireSuccess()
         }
 
-        val pkgInfos = parsePackageInfos(jsonFile).also { jsonFile.parentFile.safeDeleteRecursively() }
+        val pkgInfos = parsePackageInfosV1(jsonFile).also { jsonFile.parentFile.safeDeleteRecursively() }
```

Comment:
Why are these private functions as well renamed with a "V1" suffix? Being private in this file, these should not interfere with other V2-specific implementations from another file.